external/thorntail-mp-tck/README.md

```diff
@@ -0,0 +1,31 @@
+# External Thorntail-mp-tck Tests
+
+Thorntail-mp-tck tests are a part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Thorntail-mp-tck test material is pulled from the [thorntail-mp-tck](https://github.com/thorntail/thorntail) repository.
```

Comment:
Please change Thorntail-mp-tck tests
to 
Thorntail Microprofile TCK tests
at the start of the sentence

Reply:
and please change the end of the sentence to read:
pulled from the thorntail-mp-tck repository
to 
pulled from the thorntail repository 
since that is the name of the repository we are pulling from

Reply:
Alright, working on it ASAP.

Reply:
Should the heading "External Thorntail-mp-tck Tests" be changed as well?

Reply:
Yes, please do, thanks!

Reply:
Alright!

Reply:
The required changes has been made. Kindly review.
Thank you!